server-example/app.py

```diff
@@ -0,0 +1,26 @@
+# Example CI server that serves badges
+
+from flask import Flask
+import pybadges
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def serveBadges():
+    # First example
+    badge_arg = dict(
+        left_text='build',
+        right_text='passing',
+        right_color='#008000'
+    )
+    badge = pybadges.badge(**badge_arg)
+
+    # Second example
+    secondBadge = pybadges.badge('chat', 'online')
+
+    return badge + "\n" + secondBadge
```

Comment:
I think that you should serve the images as images. You probably need to only serve one badge and do something like:
```python
        response = flask.make_response(badge)
        response.content_type = 'image/svg+xml'
        return response
```